### DISCUSSION TITLE
After IOS 15 upgrade no response

### DISCUSSION THREAD

**Discussion by PhilBrien** [HomeSpan/HomeSpan#164] [Q&A] [answered]
I upgraded my homepod minis iPhone and iPads a few days ago to IOS15 and now, a few hours after reboot my homespan devices all come up with no response in the home app. I have a led lamp and a temperature and humidity sensor that are doing the same thing. Any help with tracking down why this is happening would be greatly appreciated. Prior to the IOS update the homespan devices worked just fine. Thanks.

**Comment by HomeSpan** [comment]
I will be upgrading to iOS15 this weekend and will see if the same issue arises.

**Comment by SnowballsChance** [comment]
I have devices on both iOS 14.x and 15.0 and they all work ok with my HomeSpan device. Seems fine when on home WiFi and when away from home on cellular data. I have an Apple TV that acts as a hub.

**Comment by PhilBrien** [comment]
I have 3 Homepod minis and 1 Apple TV all on IOS 15.0. I am going to try a simple Homespan device when my ESP32 boards arrive. Both my current Homespan devices are bridges with 2 or more accessories.

**Reply by PhilBrien** [reply]
I have several accessories now working on IOS15 do the issues are unrelated I think.

**Comment by HomeSpan** [comment]
Has the issue now been resolved, or are you still having trouble with the HomeSpan devices?

**Comment by PhilBrien** [comment, marked as answer]
I can't say what the problem was however upgrading all my devices to ios15, and improving my wifi signal strength seem to have resolved my problems. I have three devices with a combination of different accessories all working fine for over a week now.
